main.go

```diff
@@ -113,6 +114,8 @@ func defineFlagVar() *FlagVar {
 		"Indicates the failure max delay for rate limiter. .")
 	flag.DurationVar(&flagVar.reconciliationInterval, "reconciliation-interval", 1*time.Hour,
 		"Indicates the time based reconciliation interval of APIRule.")
+	flag.DurationVar(&flagVar.migrationInterval, "migration-interval", 1*time.Minute,
```

Comment:
Is there any benefit to expose this? If we want to keep it we should also adapt documentation of the parameters: https://github.com/kyma-project/istio/blob/main/docs/user/technical-reference/05-00-istio-controller-parameters.md#all-configuration-parameters.

If we don't want to keep it, I'd keep it simple and not even have it in the `ApiRuleReconcilerConfiguration`, but only have it as a const used in the `apirule_controller.go`.